https://issues.redhat.com/browse/ACM-6223 Update late binding 2.6 (#5261)
open-cluster-management/rhacm-docs: multicluster_engine/cluster_lifecycle/create_infra_env.adoc

Infrastructure environments support the following features:

- Zero-touch provisioning of clusters: Deploy clusters using a script. See https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html/scalability_and_performance/ztp-deploying-disconnected[Deploying distributed units at scale in a disconnected environment] in the {ocp} documentation for more information.

> - Late binding: Delay binding a cluster to a host. An infrastructure administrator can boot the host, and the creator of a cluster can bind the host to the exisitng cluster at a later time. The cluster creator does not need administrator privileges to access the infrastructure when using late binding.

Reviewer: The same issue in the first sentence. Should be :Delay binding a host to a cluster.
In addition, I assume the explanation can be Like:
Late binding: Enable to add additional node to the existing cluster. An infrastructure administrator can boot the host, and the creator of a cluster can bind the host to the existing cluster at a later time ...

Also - please fix the "exsitng" cluster to existing (misspell)

> - Late binding: Add a node to an existing cluster. An infrastructure administrator can boot the host, and the creator of a cluster can bind the host to the existing cluster at a later time. The cluster creator does not need administrator privileges to access the infrastructure when using late binding.

Reviewer 2: ```suggestion
- Late binding: Add a node to an existing cluster. An infrastructure administrator can boot the host, and the creator of a cluster can bind the host to the existing cluster at a later time. Administrator privileges are not required for the cluster creator to access the infrastructure when using late binding.
```

Reviewer 2: ```suggestion
- Late binding: Add a node to an existing cluster. As an infrastructure administrator, you can boot the host, and the cluster creator can bind the host to the existing cluster at a later time. Administrator privileges are not required for the cluster creator to access the infrastructure when using late binding.
```

> - Late binding: Add a node to an existing cluster. An infrastructure administrator can boot the host, and the creator of a cluster can bind the host to the existing cluster at a later time. The cluster creator does not need administrator privileges to access the infrastructure when using late binding.

Reviewer 2: ```suggestion
- Late binding: Add a node to an existing cluster. An infrastructure administrator can boot the host, and the cluster creator can bind the host to the existing cluster at a later time. The cluster creator does not need administrator privileges to access the infrastructure when using late binding.
```

> - Late binding: Add a node to an existing cluster. An infrastructure administrator can boot the host, and the creator of a cluster can bind the host to the existing cluster at a later time. Administrator privileges are not required for the cluster creator to access the infrastructure when using late binding.

Reviewer 2: ```suggestion
- Late binding: Add a node to an existing cluster. An infrastructure administrator can boot the host, and the cluster creator can bind the host to the existing cluster at a later time. Administrator privileges are not required for the cluster creator to access the infrastructure when using late binding.
```
